### DISCUSSION TITLE
"Properties" not showing in Taskbar right click context menu

### DISCUSSION THREAD

**Discussion by MightyM8** [valinet/ExplorerPatcher#1929] [Q&A]
Windows 11 build 22631.2191
Windows Feature Experience Pack 1000.22671.1000.0

When I right click on the Taskbar, the context menu is abreviated from what it was previously.
![image](https://github.com/valinet/ExplorerPatcher/assets/95981915/539238f7-e0a5-45f3-9c49-db8fe1f2dac2)

I am not able to access the Explorer Patcher settings screen.

**Comment by kwhopson** [comment]
I have this problem too, or similar rather, on three different PCs.  The difference is I'm not even able to see Taskbar Manager and Taskbar Settings. When I right-mouse on the taskbar, all icons disappear for a second, then return, but Manager, Settings or Properties for Explorer Patcher never show. Oddly, after initially installing, it did work like it is supposed to, and I was able to use the Explorer Patcher Properties to have the Start Menu show All Apps by default.  I'm not sure when right-mousing on the taskbar stopped working, but it was within a few days of installation at the most.

**Comment by pyrates999** [comment]
to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

Then go to advanced and make sure the option **Do not change the taskbar context menu (e.g. do not display the "Properties"** is not enabled.  Then you can right click on the taskbar and you should see the **Properties** option.

**Comment by chasemasters** [comment]
I did this, but now I don't see properties.  I see settings, lock task bar, task manager, etc, but not properties.

**Reply by kwhopson** [reply]
I'm not in front of my computer right now to verify but if I remember right, following pyrates999 instructions did launch  ExplorerPatcher though the Advanced option they suggested did not bring Properties back. Since I now had a way to launch EP I toyed around with more settings and found if I toggled one of the options from Windows 11 to Windows 10, it still retained all my specific Windows 11 tweaks as well as brought Properties back when right-mousing on the taskbar (this is the part I can't exactly verify, I don't recall where I toggled 11 to 10, but it was near the top, a pull-down I think). Hope that helps.

**Reply by chasemasters** [reply]
Yep, that worked.

**Reply by pyrates999** [reply]
What setting did you change so it can be marked as the answer?
